specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2019-04-01/containerregistry_build.json

```diff
@@ -817,6 +823,10 @@
           "type": "string",
           "readOnly": true
         },
+        "updateTriggerToken": {
```

Comment:
Could we move this property at the end as well?

Reply:
Sure, will do.